.github/workflows/deploy.yml

```diff
@@ -0,0 +1,47 @@
+name: Push new build
+on:
+  pull_request: null
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  deploy:
```

Comment:
NIT: Separate build and deploy jobs 🙏

Reply:
not gonna, mostly because I don't want to just have an artifact passing step when it's available right there (the deploy is quick). I'll rename this though